### ISSUE TITLE
the encode file can change the DRACO header to my own defined word?

### ISSUE THREAD

**Issue by Frankling** [google/draco#734]
the encode file can change the DRACO header to my own defined word?how to build the javascript encoder and decoder

**Comment by tomfinegan** [comment]
> the encode file can change the DRACO header to my own defined word?

Yes, but the files would no longer decode with a standard build of Draco.

> how to build the javascript encoder and decoder

Build instructions, including for the javascript tools: https://github.com/google/draco/blob/master/BUILDING.md
